### ISSUE TITLE
Add more UI screenshots to the website

### ISSUE THREAD

**Issue by yurishkuro** [jaegertracing/documentation#193]
We don't feature the UI screenshots prominently on the website, even though they provide nice visuals. E.g. currently they are below the fold inside the documentation (https://www.jaegertracing.io/docs/1.8/). Could we find a place on the home page?

cc @tiffon

**Comment by yurishkuro** [comment]
cc @lucperkins @tiffon for ideas

**Comment by pavolloffay** [comment]
We should add
* trace graph
* compare traces

trace view could be changed with the trace comparison buttons.

**Comment by tiffon** [comment]
I would suggest not including a ss of the trace graph view while the view is still experimental.

**Comment by 141AQUA-xzy** [comment]
How about adding a horizontal scrollbar/or pagination and it will take space of one full ss and essentially solve the problem

Leveraging us to add more/how much we want....!

**I can work upon that**

**Comment by yurishkuro** [comment]
@141AQUA-xzy I would want to see it in action before saying yes or not

**Comment by 141AQUA-xzy** [comment]
@yurishkuro Okay I understand that......

Something like this,with a modern touch to make jaeger stand tall....

I'm refering this image because it's time taking to make it all developed.....and if it goes waste then 🤥

![Image](https://github.com/user-attachments/assets/d68491b6-8980-49cb-a62e-1b6e765c0bd7)

**Comment by 141AQUA-xzy** [comment]
md your confirmation so that I could start working upon that......

**Comment by yurishkuro** [comment]
This view does not work for screenshots, it's too wide. And where on the home page would it go?

An alternative to plain screenshot is to do a screen cast, similar to https://perses.dev/

**Comment by 141AQUA-xzy** [comment]
Okay then what I suggest is.....as you suggested we'll play an screen-cast in the same amount of space as the ss is held in the current scenario........and since to make it more visual ,we'll provide a button[text : `View in Fullscreen`] or controller which will open the screen-cast in full screen.....analogous to what happens on Youtube full screen controller and once the screencast is dbclicked/or simply clicked we'll return to the normal doc view......it's possible and visually appealing.......

What it feels like ?

**Comment by yurishkuro** [comment]
Can you show a mock screenshot?

**Comment by 141AQUA-xzy** [comment]
SURE....!

See how the ss is shown resizing...with click gesture


### **Before ss Clicked**
![Image](https://github.com/user-attachments/assets/4d528e93-df7a-4664-bcf3-6c580b4bd732)

### **After Clicked : Full-screen view**
![Image](https://github.com/user-attachments/assets/aa97d50b-7c34-4cf4-b560-552d2ee04f0a)

So, we will replace this ss with a screen-cast ,it is meant to walk the user through usage guide....and the screencast once clicked, opens in full-screen view as shown in **_after_** case [ not the content but the view ]

Basically ,it will be eye-catchy,intuitive & informative at the same time.......also with these static ss the lack of click gesture is existing.......So we will attempt to facilitate the challenges with solution......

If possible,once provide me with the screencast and I'll show you how exactly I wanted to make it look like......!

#829  Please checkout this PR ,it seems working now !!

**Comment by yurishkuro** [comment]
The ticket is asking if we can add screenshots / screencast to the home page. We already have s/s in the docs, there's nothing to be done there.

**Comment by 141AQUA-xzy** [comment]
![Image](https://github.com/user-attachments/assets/e59248fc-f8e4-484a-83d5-0276806e1512) \

this page has plenty of space we can make it available

 I interpreted that earlier but thought docs ss has lack of room,so don't troll me for that...haha

If I'm not wrong the page mentioned above is the home page,the one under count....Right ?

We are assumed to add screencast/ss on Home Page ?

**Comment by yurishkuro** [comment]
That's what I was asking, a mock of how the screencast can be incorporated into the home page.

**Comment by 141AQUA-xzy** [comment]
Okay wait for a while.....

I show you....

**Comment by 141AQUA-xzy** [comment]
Just hand-over the screenplay 

![Image](https://github.com/user-attachments/assets/5a6e37c1-0d24-436e-b5e5-946485d88770)

![Image](https://github.com/user-attachments/assets/3cd68f7e-4609-4ca0-a1df-03dbb05971b2)

**Comment by 141AQUA-xzy** [comment]
@yurishkuro Sir please help.....for next steps !!
